src/transformers/modeling_prophetnet.py

```diff
@@ -261,7 +262,7 @@ class ProphetNetSeq2SeqLMOutput(ModelOutput):
 
             Attentions weights of the predict stream of the decoder, after the attention softmax, used to compute the
             weighted average in the self-attention heads.
-        decoder_cross_attentions (:obj:`tuple(torch.FloatTensor)`, `optional`, returned when ``output_attentions=True`` is passed or when ``config.output_attentions=True``):
```

Comment:
This was a bad naming choice. IMO, `cross_attentions` is better than `decoder_cross_attentions` because (similar to the naming of `past_key_values`) the encoder cannot have any `cross_attentions`.